<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trabCPD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>94</height>
     </rect>
    </property>
    <property name="title">
     <string>Sorts:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="sortA">
       <item>
        <property name="text">
         <string>Insertion Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shell Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bubble Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Heap Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Counting Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Merge Sort</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortB">
       <item>
        <property name="text">
         <string>Insertion Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shell Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bubble Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Heap Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Counting Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Merge Sort</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuração do Array:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="arrayConfig">
       <item>
        <property name="text">
         <string>Aleatório </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ordenado</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Invertido</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aleatório pré-definido</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="compareButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Comparar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
